addons/account_edi_ubl_cii/models/account_move.py

```diff
@@ -52,3 +52,9 @@ def _get_edi_decoder(self, file_data, new=False):
                 return ubl_cii_xml_builder._import_invoice_ubl_cii
 
         return super()._get_edi_decoder(file_data, new=new)
+
+    def get_edi_doc_attachments_to_export(self):
+        edi_doc_attachments = super().get_edi_doc_attachments_to_export()
+        for ubl_cii_attachments in self.ubl_cii_xml_id:
+            edi_doc_attachments.append(ubl_cii_attachments)
+        return edi_doc_attachments
```

Comment:
```suggestion
    def get_edi_doc_attachments_to_export(self):
        # EXTENDS 'account'
        return super().get_edi_doc_attachments_to_export() + self.ubl_cii_xml_id
```
This method should be private